Src/BootCamp.Chapter/ContactsCenter.cs

```diff
@@ -1,27 +1,66 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.IO;
 using System.Text;
 
 namespace BootCamp.Chapter
 {
     public class ContactsCenter
     {
-        private readonly List<Person> _people;
+         private readonly string _peopleFile;
 
         public ContactsCenter(string peopleFile)
         {
-            // load people
+            if (!File.Exists(peopleFile))
+            {
+                throw new FileNotFoundException("File is made up"); 
+            }
+                      
+            _peopleFile = File.ReadAllText(peopleFile); 
+           
+            if (String.IsNullOrWhiteSpace(_peopleFile))
+            {
+                throw new Exception(); 
```

Comment:
What is this exception for? No details just "something went wrong". Not going to be useful when debugging an stack trace. 

Tend to never want to just throw `Exception` but a specific type. `InvalidOperationException` might be better here because that suggests that something is not in the right state for this path.

Also, a message would be good.
Either

Reply:
thanks for the remark
I choose Exception because of this tests "
```
 public void New_With_Empty_File_Throws()
        {
            const string path = "Input/Empty.txt";

            Action action = () => new ContactsCenter(path);

            // Leaving this up to a student to figure out.
            action.Should().Throw<Exception>();
        }
````

otherwise I would totally agree with you

Reply:
He, when I change it to a InValidOperationException and give it a message the test are still green. Can a mentor exaplian why ? Or is it that the InvalidOperationException inherits from Exception that ir works.